### DISCUSSION TITLE
Unable to import or export pages or collections

### DISCUSSION THREAD

**Discussion by lampy255** [outline/outline#5665] [Self Hosting] [answered]
**To Reproduce**
1. Host Outline 0.70.2 in docker from outlinewiki/outline
2. Create a few simple pages/collections
3. Preferences > Export > Export Data > Markdown (or JSON)

**Expected behavior**
Successful export as .zip archive

**Screenshots**
<img width="860" alt="Screenshot 2023-08-08 at 10 18 47 PM" src="https://github.com/outline/outline/assets/67034721/9b3506a3-c524-4db5-a10b-3b3671a90e4d">
<img width="860" alt="Screenshot 2023-08-08 at 10 18 35 PM" src="https://github.com/outline/outline/assets/67034721/4f3a04cb-bcac-4cdd-b0a0-7793983316c9">

** Log Dump **
[FailedExport1.txt](https://github.com/outline/outline/files/12290150/FailedExport1.txt)

** Notes **
Everything else is working fine. Minio storage (RELEASE.2023-08-04T17-40-21Z.hotfix.04968f7ec)
No problem uploading images etc  

- NGINX Proxy manager
- Redis
- Postgres

**Outline:**
- Install: self hosted
- Version: b9fc301 (0.70.2)

**Desktop:**
 - OS: MacOS Ventura 13.4.1
 - Browser: Google Chrome
 - Version: 115.0.5790.170

**Comment by tommoor** [comment, marked as answer]
The outline container cannot reach the storage container, check DNS – it's almost always DNS
